strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/testing/artifact/TestArtifactContext.java

```diff
@@ -209,31 +211,18 @@ private void store(Map.Entry<RepositoryPath, Path> e)
         {
             throw new UndeclaredThrowableException(ioe);
         }
-        
-        try
-        {
-            Files.delete(e.getValue());
-        }
-        catch (IOException ioe)
-        {
-            throw new UndeclaredThrowableException(ioe);
-        }
+
+        ThrowingConsumer.unchecked(Files::delete).accept(e.getValue());
```

Comment:
@adavid9 can you please add `throws IOException` and use `ThrowingConsumer` within `TestArtifactContext.deployArtifact()` foreach